### PR TITLE
[dom] CDO: Version and toolchain update (PE 17.08)

### PR DESCRIPTION
**Pull request by omlins**

### CODE REVIEW

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ scientific data."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.unidata.ucar.edu/downloads/netcdf/ftp/']
+
+dependencies = [
+    ('cray-hdf5/1.10.0', EXTERNAL_MODULE),
```
Comment (jgphpc) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb]: `cray-hdf5/1.10.0` conflicts with `cray-hdf5/1.10.0.3`
in CDO-1.9.0-CrayGNU-17.08.eb ?

**Comment by jgphpc** [comment]
git checkout craype/17.08 -- easybuild/easyconfigs/c/cURL/cURL-7.55.1-CrayGNU-17.08.eb

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+]
+
+postinstallcmds = [('ln -s %(installdir)s/lib64 %(installdir)s/lib')]
+
+builddependencies = [
+    ('CMake', '3.9.1', '', True),
+    #('Doxygen', '1.8.9.1', '', True),
+    ('cURL', '7.49.1', '', True),
```
Comment (jgphpc) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb]: `Irresolvable dependencies encountered: cURL/.7.49.1` => need to push it

Comment (jgphpc) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb]: https://github.com/eth-cscs/production/blob/craype/17.08/easybuild/easyconfigs/c/cURL/cURL-7.55.1-CrayGNU-17.08.eb

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb (hunk 1 above)
Comment (omlins) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb]: Done.

File: easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb (hunk 2 above)
Comment (omlins) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb]: Done.

**Pull request merged**